packages/client/src/utils.ts

```diff
@@ -0,0 +1,38 @@
+import { IDToken } from './types';
+
+const fullfillBase64 = (input: string) => {
+  if (input.length === 2) {
+    return `${input}==`;
+  }
+
+  if (input.length === 3) {
+    return `${input}=`;
+  }
+
+  return input;
+};
+
+/**
+ * Decode IDToken from jwt, without verifing.
+ * Verifing jwt requires fetching public key first, this can not
+ * be done in a sync function, in some cases, verifing is not necessary.
+ *
+ * @param token jwt string
```

Comment:
let's use `JWT` for cross-repo naming convention

Reply:
also let's follow the same format for docs
![image](https://user-images.githubusercontent.com/14722250/137098647-ab7428c9-01ae-49a0-ac56-d5f2009c2b57.png)